charter.md

```diff
@@ -0,0 +1,104 @@
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+- Production of high quality Kubernetes releases on a reliable schedule
+- Ensure there is a consistent group of community members in place to support the release process
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to empower SIGs to integrate their repositories into the release process
+
+### In scope
+
+- Ensuring high quality Kubernetes releases
+  - Define and staff release roles to manage the resolution of release blocking criteria
+  - Define and drive development processes (e.g. merge queues, cherrypicks) and release processes 
+    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
+  - Manage the creation of release specific artifacts including
+    - Code branches
+    - Binary distributions
+    - Release notes
+- Continually improving release and development processes
+  - Work closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
+  - Work with downstream communities responsible for packaging Kubernetes releases
+  - Work with other SIGs to agree upon the responsibilities of their SIG with respect to the release
+  - Define and collect metrics related to the release in order to measure progress over each release
+- Collaboration with downstream communities which build artifacts from Kubernetes releases
+  - Ensure the appropriate level of integration with publishing the OSS build artifacts
+
+### Out of scope
+
+#### Support
+
+SIG Release itself is not responsible for end user support or creation of patches for support streams. There are support forums for end users to ask questions and report bugs, subject matter experts in other SIGs triage and address issues and when necessary mark bug fixes for inclusion in a patch release.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+Specifically, the common guidelines (see: [sig-governance]) for continuity of membership within roles in the SIG are followed.
+
+### Deviations from [sig-governance]
+
+- SIG Release does not have top-level SIG Technical Leads; SIG Release Chairs can serve in both an operational and technical capacity
+- SIG Release subprojects have subproject chairs
+
+#### SIG Membership
+
+Additionally, SIG Release has a concept of membership. SIG members can be occasionally called on to assist with decision making, especially as it relates to gathering historical context around existing policies.
+
+While a strict definition of membership has yet to be established, we can initially seed a set of members from the list of Release Team members from Kubernetes 1.10 through to Kubernetes 1.13.
```

Comment:
IMO this should be all documented former release team members.

Reply:
Yep. That sounds good. I think my hesitancy around saying that was having to build the list.
How can we ensure that that list remains fresh i.e., active SIG Release members as opposed to the entire historical set? That also raises the question of how we classify an active member. 

I was also thinking of adding a `sig-release-members` entry in `OWNERS_ALIASES`. Thoughts?

Reply:
@justaugustus since OWNERS_ALIASES is for review/approvals, we should may be just use a github group. would be easy to add/drop people too.

Reply:
+1 to `all documented former release team members`